src/common/fileList/fileList.module.scss

```diff
@@ -0,0 +1,22 @@
+@import 'colours';
+@import 'fonts';
+@import 'spacings';
+
+.markedForDeletion {
+  color: $gray-500;
+}
+
+.fileListItem {
+  margin: units(0.25) 0;
+  display: flex;
+  align-items: center;
+}
+
+.fileName {
+  flex-grow: 1;
+  line-height: font-size(24);
```

Comment:
We are not setting font-size here so could we just write `24px` ?

Reply:
We're trying to avoid setting sizes outside the standard spacings, but good catch. `units(1.5)` gives the same result and makes more sense than `font-size(24)`